### PR TITLE
Delete Services from a NonProfits Database 

### PR DESCRIPTION
**Pull request by JackCampbell5**
## Description

<what this pull request is doing>
<what will be done in later PRs and not included here>

## Milestones
+ Stretch Feature 

## Resources
+ [Live Site](https://strong-start-git-np-delete-jack-campbells-projects.vercel.app/)

## Test Plan
<insert images or gifs of feature>
<otherwise, say how code was tested if not UI facing>
<any edge cases you might have specifically tested>

### CODE REVIEW

File: frontend/src/components/RecList/RecList.jsx [hunk 1]
```diff
@@ -56,7 +56,12 @@ function RecList({ data }) {
    * + This will show a success message for 5 seconds and no longer show the service
    * @param {string} id - The id of the service that was added
    */
-  function serviceAddedSuccessfully(id, validData = {}, editOnly = false) {
+  function serviceAddedSuccessfully(
```
Comment (ShauryaS) [inline frontend/src/components/RecList/RecList.jsx:59]: rename this to something that encompasses that this adds and removes services

**Review by ShauryaS — APPROVED** [review]
LG. Please resolve the minor comment before merge

File: frontend/src/components/RecList/RecList.jsx (hunk 1 above)
Comment (JackCampbell5) [reply frontend/src/components/RecList/RecList.jsx]: Updated: https://github.com/JackCampbell5/strong-start/pull/77/commits/7b2107f858a47fa4c915dfa008915e6ff401c39d
